src/components/settingFlyout/settingFlyout.js

```diff
@@ -87,6 +87,9 @@ class SettingFlyout extends React.Component {
     return (
       <div className="setting-workflow-container">
         <div className="setting-section">
+          <div className="section-header">{ 'Version 0.0.1' }</div>
```

Comment:
0.0.1?

Reply:
Placeholder value, cleared with Hector.

Reply:
I would like to use the tag from the top level repos as the version number.  For Monday this will be `1.0.0-preview.3`.  Can you change it to this?